include/swift/Demangling/StandardTypesMangling.def

```diff
@@ -88,7 +88,7 @@ STANDARD_TYPE_CONCURRENCY(Protocol, F, Executor)
 STANDARD_TYPE_CONCURRENCY(Protocol, f, SerialExecutor)
 STANDARD_TYPE_CONCURRENCY(Structure, G, TaskGroup)
 STANDARD_TYPE_CONCURRENCY(Structure, g, ThrowingTaskGroup)
-STANDARD_TYPE_CONCURRENCY(Protocol, h, _TaskExecutor)
```

Comment:
I just picked `h` here since seems to be available -- could I ask someone to double check that's fine?

@DougGregor @hborla @rjmccall